src/boxrobots/test/visualize.jl

```diff
@@ -7,6 +7,9 @@
   vis_options = br.BoxRobotVisualizerOptions(force_arrow_normalizer=mass*abs(gravity[end]))
 
   DrakeVisualizer.any_open_windows() || DrakeVisualizer.new_window();
+  println("waiting for visualizer to start up")
+  sleep(5) # let visualizer start up
```

Comment:
oh yeah, I should make a way to wait for the vis to come up reliably. Issue opened: https://github.com/rdeits/DrakeVisualizer.jl/issues/46